CONTRIBUTING.md

```diff
@@ -58,6 +58,13 @@ Our code review guidelines are documented [here](https://audacia.co.uk/technical
 
 Before submitting a pull request, please ensure that the changes have been documented in the relevant `CHANGELOG.md`
 
+### Versioning
+
+Audacia libraries previously utilised a versioning system that would automatically iterate the patch numbers in instances where neither the major or minor number had been altered and determined if a package was to be marked as prerelease based on the branch name (a name other than `main` or `master`).
+This functionality has been removed in favour of a manual system where the branch is tagged and released under whatever version number has been entered in either the CSPROJ or package.json.
+Update version numbers according to the significance of the change being added.
```

Comment:
`nit`: I think this should go first. i.e., Tell people what to do, then justify it.